website/ReadingAndWritingImageFiles.rst

```diff
@@ -9,7 +9,7 @@ Document Purpose and Audience
 =============================
 
 This document shows how to write C++ code that reads and writes OpenEXR
-2.0 image files.
+3.0 image files.
```

Comment:
The 2.0 is actually correct here, since it refers to version 2 of the file format, not of the library release, although the text should make that clear, it is indeed confusing.

Reply:
My mistake! I will revert this change. 
I also get the same build failures with the use of `clamp`, I will take a look and see if using `std::clamp` gets rid of them. Taking a look at the other comments as well.